modules/ml/src/test/resources/mnist-config.xml

```diff
@@ -0,0 +1,21 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
```

Comment:
MB this config from local tests? in that case it should be deleted too.

Reply:
Yes, this is legacy from my early experiments.